config.js

```diff
@@ -70,7 +70,13 @@ Config.prototype._seed = function _seed(seed) {
     seedOrDefault('dampScoringReuseLimit', 2500);
     seedOrDefault('dampScoringSuppressDuration', 60 * 60 * 1000); // 1 hr in ms
     seedOrDefault('dampScoringSuppressLimit', 5000);
-    seedOrDefault('maxJoinAttempts', 50, numValidator);
+
+    // Joiner config
+    seedOrDefault('joinDelayMin', 100, numValidator); // ms
+    seedOrDefault('joinDelayMax', 2 * 60 * 1000, numValidator); // 2 min in ms
+    seedOrDefault('joinTroubleErrorEnabled', true);
+    seedOrDefault('maxJoinDuration', 20 * 60 * 1000, numValidator); // 20 mins in ms
```

Comment:
agreed, but I did this to preserve backwards compatibility as there is an option that could have been provided to Ringpop in previous releases that would have set the maxJoinDuration. Keeping it named as such will retain that value and not break existing apps.